pallets/asset-manager/src/migrations.rs

```diff
@@ -28,9 +28,9 @@ use sp_std::vec::Vec;
 use xcm::latest::prelude::*;
 
 /// Migration that changes the mapping AssetId -> units_per_second to
-/// a mapping of the form AssetType -> units_per_second
-/// It does so by removing the AssetTypeUnitsPerSecond storage and
-/// populating the new AssetTypeUnitsPerSecond
+/// a mapping of the form ForeignAssetType -> units_per_second
+/// It does so by removing the ForeignAssetTypeUnitsPerSecond storage and
+/// populating the new ForeignAssetTypeUnitsPerSecond
```

Comment:
Do we need this migration at this point? Might be better to remove it if these changes are simply to make it compile...

Reply:
For now I will keep it, we can evaluate in future PRs.

Reply:
I created an issue to potentially remove them in the future